.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,76 @@
+on:
+  pull_request:
+    #types:
+    #  - labeled
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+    #if: ${{ github.event.label.name == 'TODO' }}
+    env:
+      KUBECONFIG: ${{ github.workspace }}/kubeconfig.yaml
+    steps:
+      #- name: Remove label
+      #  uses: andymckay/labeler@master
+      #  with:
+      #    remove-labels: "TODO"
+
+      # install and login
+      - uses: actions/checkout@v3
+      - name: Install Telepresence
+        uses: ./install
+      - name: Login
+        uses: ./login
+        with:
+          telepresence_api_key: ${{ secrets.TELEPRESENCE_API_KEY }}
+
+      # we need a demo cluster
+      - uses: azure/setup-kubectl@v3
+        with:
+          version: "v1.25.3"
+        id: kubectl
+      - name: Provision Kubeception cluster
+        uses: datawire/infra-actions/provision-cluster@v0.2.0
```

Comment:
is there a reason not to use the last version ?

Reply:
not really, i just copied from our integration tests w/o checking